src/main/java/com/adobe/pdfservices/operation/samples/exportpdftoimages/ExportPDFToJPEGZip.java

```diff
@@ -43,17 +48,23 @@ public static void main(String[] args) {
 
             //Create an ExecutionContext using credentials and create a new operation instance.
             ExecutionContext executionContext = ExecutionContext.create(credentials);
-            PDFPropertiesOperation pdfPropertiesOperation = PDFPropertiesOperation.createNew();
+            ExportPDFToImagesOperation exportPDFToImagesOperation = ExportPDFToImagesOperation.createNew(ExportPDFToImagesTargetFormat.JPEG);
+
+            // Set operation input from a source file.
+            FileRef sourceFileRef = FileRef.createFromLocalFile("src/main/resources/exportPDFToImagesInput.pdf");
+            exportPDFToImagesOperation.setInput(sourceFileRef);
+
+            // Set the output type to create zip of images
+            exportPDFToImagesOperation.setOutputType(OutputType.ZIP_OF_PAGE_IMAGES);
 
-            // Provide an input FileRef for the operation
-            FileRef source = FileRef.createFromLocalFile("src/main/resources/pdfPropertiesInput.pdf");
-            pdfPropertiesOperation.setInputFile(source);
+            // Execute the operation.
```

Comment:
Let us either keep a full stop everywhere in the file comments, or remove it from everywhere in the file comments.